packages/react/src/react/views/collections/list/list.client.tsx

```diff
@@ -69,7 +69,7 @@ export function ClientCollectionListView() {
 
     const columnHelper = createColumnHelper<BaseData>()
     return [
-      ...(listViewProps.actions?.delete
+      ...(listViewProps.actions?.select
```

Comment:
it mean `delete` will not be usable if `select` is not turned on.

```suggestion
      ...((listViewProps.actions?.select || listViewProps.actions?.delete)
```